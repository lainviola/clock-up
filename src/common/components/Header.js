import styled from 'styled-components'
import { borderColor } from '../CSS'

export const Header = styled.div`
  padding: 0 30px;
  grid-area: header;
  border-bottom: 1px solid ${borderColor};
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: flex-end;
`
