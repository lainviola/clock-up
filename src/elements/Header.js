import styled from 'styled-components'
import { theme } from '../const'

export const Header = styled.div`
  padding: 0 30px;
  grid-area: header;
  border-bottom: 1px solid ${theme.borderColor};
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: flex-end;
`
