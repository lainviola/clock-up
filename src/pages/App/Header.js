// @flow
import React from 'react'
import { pure } from 'recompose'
import styled from 'styled-components'
import { theme } from '../../theme'
import LogoutButton from './LogoutButton'

export const StyledHeader = styled.div`
  padding: 0 30px;
  grid-area: header;
  border-bottom: 1px solid ${theme.borderColor};
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: flex-end;

  @media (max-width: 767px) {
    display: none;
  }
`

const Header = () => (
  <StyledHeader>
    <LogoutButton />
  </StyledHeader>
)

export default pure(Header)
