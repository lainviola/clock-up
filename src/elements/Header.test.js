import React from 'react'
import { shallow } from 'enzyme'
import { Header } from './Header'

describe('<Header />', () => {
  it('should match to spapshot', () => {
    const wrapper = shallow(<Header />)
    expect(wrapper).toMatchSnapshot()
  })
})
