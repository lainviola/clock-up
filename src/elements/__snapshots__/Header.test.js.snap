// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should match to spapshot 1`] = `
<div
  className="Header-bKPLEa eAZYFB"
/>
`;
